homeassistant/components/calendar/__init__.py

```diff
@@ -86,15 +86,19 @@ def calculate_offset(event, offset):
     summary = event.get("summary", "")
     # check if we have an offset tag in the message
     # time is HH:MM or MM
-    reg = "{}([+-]?[0-9]{{0,2}}(:[0-9]{{0,2}})?)".format(offset)
+    reg = "{}(([+-]?[0-9]{{0,2}}:[0-9]{{0,2}})|([+-]?[0-9]{{1,3}}))".format(offset)
```

Comment:
This change doesn't look correct to me. MM should only be 0-59. Is there any docs about how the offset should be specified?

Reply:
The whole point of this change was to allow 60+ minutes to be defined. The documentation is a bit fuzzy about how this is defined:

> A set of characters that precede a number in the event title for designating a pre-trigger state change on the sensor. This should be in the format of HH:MM or MM.

Using the format MM, I would expect that I could use any number of minutes, not only 0-59. I would say using 90minute offset should be perfectly fine.